lowtime/solver.py

```diff
@@ -358,279 +363,6 @@ def reduce_durations(
 
         return cost_change
 
-    def find_min_cut(self, dag: nx.DiGraph) -> tuple[set[int], set[int]]:
```

Comment:
I would actually keep this part, because someone might want to tweak the algorithm for research, and doing it on Python is easier. Could you instead note on the docstring that this has been accelerated `rust_runner.find_min_cut` but is left here for reference?